metaflow/__init__.py

```diff
@@ -108,7 +108,11 @@ class and related decorators.
 JSONType = JSONTypeClass()
 
 # data layer
-from .datatools import S3
+# For historical reasons, we make metaflow.plugins.datatools accessible as
+# metaflow.datatools. S3 is also a tool that has historically been available at the
+# TL so keep as is.
+lazy_load_aliases({"metaflow.datatools": "metaflow.plugins.datatools"})
```

Comment:
should we move the method lazy_load_aliases out of `extension_support` now?

Reply:
and can we start expanding TL as top level here and every where else? trips me up every time.

Reply:
I'll expand TL. I don't know if we should move it out though; I've tried to keep everything related to extensions (aliasing basically) in one place.